README.adoc

```diff
@@ -225,6 +202,8 @@ to ensure that they are different from each other.
 
 The `@Test` annotation indicates that the method automatically executes when your test class runs.
 
+The TestUtils class includes utility methods to make connection to the services and automate property changes.
```

Comment:
It looks like `TestUtils` should be formatted as a code phrase.

Also what if we added the two words in bold below?
"The `TestUtils` class includes utility methods to make **a** connection to the services and **to** automate property changes."

Reply:
fixed